docs/setup/yarn_setup.md

```diff
@@ -100,11 +100,13 @@ Usage:
      -D <arg>                        Dynamic properties
      -d,--detached                   Start detached
      -jm,--jobManagerMemory <arg>    Memory for JobManager Container [in MB]
+     -nm,--name                      Set a custom name for the application on YARN
```

Comment:
Fair point. I don't think it's necessary but if it is for the sake of user satisfaction you have my +1.